modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -11,8 +11,8 @@ After users xref:exchange::to-request-access.adoc[request access] to your API in
 
 To manually approve or reject requests, you must have:
 
-* The Organization Administrator or Manage Contracts permission
 * Configured the SLA tier for the application 
+* The Organization Administrator or Environment Administrator roles, or the Manage Contracts permission
```

Comment:
`Environment administrator` is also a permission in Access Management, not a role 
![image](https://user-images.githubusercontent.com/12700293/129949288-262f0091-1d4d-45fb-9fa6-3ba40169d92e.png)